### ISSUE TITLE
ERROR: parsers/onnx/ModelImporter.cpp:163 In function parseGraph: [6] Invalid Node - /ArgMax cannot create std::vector larger than max_size()

### ISSUE THREAD

**Issue by naveenkumarkr723** [NVIDIA/TensorRT#2534]
## Description

trtexec --onnx=/workspace/data/NaveenJadi/ParSeq/onnx-simplifier/parseq_ref_sim.onnx  --saveEngine=parseq_simple.engine --exportProfile=parseq_simple.json --separateProfileRun > parseq_simple.log --int8
[12/08/2022-08:01:19] [W] [TRT] parsers/onnx/onnx2trt_utils.cpp:367: Your ONNX model has been generated with INT64 weights, while TensorRT does not natively support INT64. Attempting to cast down to INT32.
[12/08/2022-08:01:19] [W] [TRT] parsers/onnx/onnx2trt_utils.cpp:395: One or more weights outside the range of INT32 was clamped
[12/08/2022-08:01:19] [W] [TRT] Tensor DataType is determined at build time for tensors not marked as input or output.
[12/08/2022-08:01:19] [E] Error[3]: /ArgMax: at least 2 dimensions are required for input.
[12/08/2022-08:01:19] [E] [TRT] parsers/onnx/ModelImporter.cpp:773: While parsing node number 614 [ArgMax -> "/ArgMax_output_0"]:
[12/08/2022-08:01:19] [E] [TRT] parsers/onnx/ModelImporter.cpp:774: --- Begin node ---
[12/08/2022-08:01:19] [E] [TRT] parsers/onnx/ModelImporter.cpp:775: input: "/Squeeze_output_0"
output: "/ArgMax_output_0"
name: "/ArgMax"
op_type: "ArgMax"
attribute {
  name: "axis"
  i: -1
  type: INT
}
attribute {
  name: "keepdims"
  i: 0
  type: INT
}
attribute {
  name: "select_last_index"
  i: 0
  type: INT
}

[12/08/2022-08:01:19] [E] [TRT] parsers/onnx/ModelImporter.cpp:776: --- End node ---
[12/08/2022-08:01:19] [E] [TRT] parsers/onnx/ModelImporter.cpp:778: ERROR: parsers/onnx/ModelImporter.cpp:163 In function parseGraph:
[6] Invalid Node - /ArgMax
cannot create std::vector larger than max_size()
[12/08/2022-08:01:19] [E] Failed to parse onnx file
[12/08/2022-08:01:19] [E] Parsing model failed
[12/08/2022-08:01:19] [E] Failed to create engine from model or file.
[12/08/2022-08:01:19] [E] Engine set up failed

## Environment

**TensorRT Version**: 8.4.1.5-1
**NVIDIA GPU**:  tensorrt
**NVIDIA Driver Version**: 
**CUDA Version**: cuda11.6 
**CUDNN Version**: 8.6
**Operating System**: Linux
**Python Version (if applicable)**: 3.8.13
**Tensorflow Version (if applicable)**:  No
**PyTorch Version (if applicable)**:  '1.13.0a0+08820cb'
**Baremetal or Container (if so, version)**: No

  Please include:
  * ParSeq github code i tried it TO convert into ONNX MODEL and then with onnx_simpler I converted ONNX_SIMPLE model 
  * but when i tried to convert it into TRT Model it gives above error , i wasnot know what is the actual problem is 

*trtexec --onnx=/workspace/data/NaveenJadi/ParSeq/onnx-simplifier/parseq_ref_sim.onnx  --saveEngine=parseq_simple.engine --exportProfile=parseq_simple.json --separateProfileRun > parseq_simple.log --int8
this was the command i used for conversation into TRT

**Comment by zerollzeng** [comment]
How does Argmax looks like, what it mean is when parsing the argmax layer, in https://github.com/onnx/onnx-tensorrt/blob/7583da4c62475e84b7be31f4b8fb0c101873d434/onnx2trt_utils.cpp#L58. it fails due to your argmax layer exceed some limitation.

**Comment by zerollzeng** [comment]
Or can you provide the onnx model here?

**Comment by naveenkumarkr723** [comment]
First i clone --> https://github.com/baudm/parseq.git  to my folder 
=========================================================================

in this read.py is there we need to give commands like ---> (some dependency like requirements.txt to be installed )
python read.py pretrained=parseq --images demo_images/roi.jpg

now i need to convert this into ONNX for that -----> I used the below code  (some dependency like requirements.txt to be installed )
================================================================================
import torch
parseq = torch.hub.load('baudm/parseq', 'parseq', pretrained=True, refine_iters=0).eval()
dummy_input = torch.rand(1, 3, *parseq.hparams.img_size)  # (1, 3, 32, 128) by default

# To ONNX
------------------------------------
parseq.to_onnx('parseq_ref2.onnx', dummy_input, opset_version=14)  # opset v14 or newer is required


then I cloned ---> https://github.com/daquexian/onnx-simplifier.git
to convert the ONNX to ONNX_SIMPLER 
================================================================================

pip3 install -U pip && pip3 install onnxsim

onnxsim parseq_ref2.onnx parseq_ref2_simple.onnx (any name u can give for storing simplified ONNX )

this created the ONNX_SIM model which i need to Transform into TRT Model 
size of Model is large so not able to share you

**Comment by zerollzeng** [comment]
Can you upload it to GDrive or create a Github repo and upload it as git LFS file?

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, please reopen if you still have question, thanks!
